lexicons/com/atproto/admin/defs.json

```diff
@@ -93,16 +89,35 @@
     },
     "actionReversal": {
       "type": "object",
-      "required": ["reason", "createdBy", "createdAt"],
+      "required": ["createdBy", "createdAt"],
       "properties": {
-        "reason": { "type": "string" },
+        "comment": { "type": "string" },
         "createdBy": { "type": "string", "format": "did" },
         "createdAt": { "type": "string", "format": "datetime" }
       }
     },
+    "actionMeta": {
+      "type": "object",
+      "properties": {
+        "resolveReportIds": { "type": "array", "items": { "type": "integer" } },
+        "reportType": {
+          "type": "ref",
+          "ref": "com.atproto.moderation.defs#reasonType"
+        }
+      }
+    },
     "actionType": {
       "type": "string",
-      "knownValues": ["#takedown", "#flag", "#acknowledge", "#escalate"]
+      "knownValues": [
+        "#takedown",
+        "#flag",
+        "#acknowledge",
+        "#escalate",
+        "#comment",
+        "#label",
```

Comment:
As an example, if a moderator adds a comment and a label at the same time then clicks "save", what event(s) are generated, and are those two events bundled together in any way?

Reply:
Referencing our previous conversations on this: we decided that this not blocking at the moment and we are settling for 1 event at a time model for now. Soon after release, if we see this being a common case, we could either end up adjusting the client to dispatch multiple requests through unified UI/UX or allow this endpoint to receive multiple events as array and process all in one go.